### PR TITLE
Increase video-rotation test tolerance, add vp9 version.

### PR DESCRIPTION
**Pull request by dalecurtis**
Speed focused software yuv conversion libraries may be off by more
than 5 points in a channel for values near the channel maximum.

There's also no reason we can't have a vp9 version of this test, so
add encodes for clients who want to test w/o proprietary codecs.

### CODE REVIEW

File: sdk/tests/conformance/textures/misc/video-rotation.html [hunk 1]
```diff
@@ -89,11 +89,15 @@
                 return;
             }
 
-            const type = 'video/mp4; codecs="avc1.42E01E, mp4a.40.2"';
-            if (!video.canPlayType(type).replace(/no/, '')) {
-                debug(type + " unsupported");
-                return;
-            };
+            let prefer_vp9 = false;
```
Comment (kenrussell) [inline sdk/tests/conformance/textures/misc/video-rotation.html:92]: Please have the test test each video format if it's supported. Thanks.

**Review by kenrussell — CHANGES_REQUESTED** [review]
Thanks for this update. One comment then lgtm.

File: sdk/tests/conformance/textures/misc/video-rotation.html (hunk 1 above)
Comment (dalecurtis) [reply sdk/tests/conformance/textures/misc/video-rotation.html]: Done.

File: sdk/tests/conformance/textures/misc/video-rotation.html [hunk 2]
```diff
@@ -141,9 +138,14 @@
...
-                fn += (prefer_vp9 ? ".vp9" : "" ) + ".mp4";
-                await testVideoElement(fn);
+            if (supports_h264) {
+                for (let fn of filenames)
+                    await testVideoElement(fn + ".mp4");
+            }
+
+            if (supports_vp9) {
+                for (let fn of filenames)
+                    await testVideoElement(fn + ".vp9.mp4");
```
Comment (kenrussell) [inline sdk/tests/conformance/textures/misc/video-rotation.html:148]: .vp9?

**Review by kenrussell — CHANGES_REQUESTED** [review]
Thanks very much for the update. One more typo?

File: sdk/tests/conformance/textures/misc/video-rotation.html (hunk 2 above)
Comment (dalecurtis) [reply sdk/tests/conformance/textures/misc/video-rotation.html:148]: No, this looks right? Sorry if I'm blind.

Comment (kenrussell) [reply sdk/tests/conformance/textures/misc/video-rotation.html:148]: Ah ok, sorry, I didn't look carefully and didn't realize these were still in an .mp4 container.

**Review by kenrussell — APPROVED** [review]
Sorry for my confusion! Thanks, looks good!

**Pull request merged**